osx_utisl.mm

```diff
@@ -0,0 +1,17 @@
+#include "osx_utils.h"
+#include <Cocoa/Cocoa.h>
+
+float getBackingScaleFactor() {
+    float scaleFactor = 1.0;
+    
+    @autoreleasepool {
+        NSArray<NSScreen *> *screens = [NSScreen screens];
+        if ([screens count] > 0) {
+            NSScreen *screen = [screens objectAtIndex:0];
+            scaleFactor = [screen backingScaleFactor];
+        }
+    }
+    
+    return scaleFactor;
+}
+
```

Comment:
Is there any way to do this without adding objective C code. I would prefer not to have to update the build system just for this issue.

Can we just call GetWindowScaleDPI in raylib?

Reply:
Thank you for the feedback. I experimented with GetWindowScaleDPI, and it appears to behave similar/identical the functionality of my current implementation. I'll continue with GetWindowScaleDPI API.